pagopa/io-app
# fix: [IOPID-2132] Centring the name of the user in the identification modal

## List of changes proposed in this pull request
- Centring the name of the user in the identification modal 

<p>

| Old screen (not centered) | New screen (centered) |
| - | - |
| <img width="250" alt="Screenshot 2024-08-07 alle 16 38 29" src="https://github.com/user-attachments/assets/645f2166-bfb1-48cb-ab72-ac56e1ad5843"/> | <img width="250" alt="Screenshot 2024-08-07 alle 16 35 21" src="https://github.com/user-attachments/assets/910a052c-7070-4157-9358-4b3eb86f34ea"> |

</p>

## How to test
Run the application, close the app and re open it. You will see this screen. 
If you want to see fastly if this modification works, you can change the name at [this line
](https://github.com/pagopa/io-app/blob/906c5efc713bfab0230de676739a2b66d10dbb44/ts/screens/modal/IdentificationModal.tsx#L232) and use some long name
